docs/docsite/requirements.txt

```
@@ -1,13 +1,18 @@
-#pip packages required to build docsite
+# pip packages required to build docsite
+# these requirements are as loosely defined as possible
+# if you want known good versions of these dependencies
+# use known_good_reqs.txt instead 
+
+antsibull >= 0.25.0
+docutils == 0.16 # pin for now until the problem with unordered lists is fixed
+# see https://github.com/readthedocs/sphinx_rtd_theme/issues/1115jinja2==3.0.1
 jinja2
+Pygments >= 2.4.0
 PyYAML
 rstcheck
-sphinx==2.1.2
+sphinx
 sphinx-notfound-page >= 0.6
 sphinx-intl
 sphinx_ansible_theme === 0.6.0
```

With https://github.com/ansible/ansible/pull/75059 it's now
```suggestion
sphinx_ansible_theme >= 0.7.0
```

---

I would really avoid `>=` here.

---

I think it's okay for direct unpinned deps. Anybody wanting the pins should do `pip install -r docs/docsite/requirements.txt -c docs/docsite/known_good_reqs.txt`